### PR TITLE
Remove old webinars from listing page and update redirect for April 9th workshop

### PR DESCRIPTION
**Pull request by zchase**
This PR adds the`unlist` to webinars that already happened this week. It also adds a redirect for the April 9th workshop to the landing page for the On-Demand recording of the workshop.

### CODE REVIEW

File: content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ meta_desc: "In this workshop, Luke Hoban shows you the easy way to start using I
 featured: false
 
 aliases: [ "/events/workshop-san-francisco-2020-04-09" ]
+redirect_to: "/webinars/getting-started-with-infrastructure-as-code/"
```
Comment (cnunciato) [inline content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md:10]: This 404s in the Pulumify preview. Should it?

Comment (justinvp) [inline content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md:10]: Should this be:

```suggestion
redirect_to: "/webinars/getting-started-with-infrastructure-as-code/on-demand/"
```

?

Also, unrelated to this PR, but I noticed it when looking at this, so I'll mention it here... on this page: https://www.pulumi.com/webinars/getting-started-with-infrastructure-as-code/on-demand/

The YouTube video shows the presenters are Luke, Cameron, and Mike, but in the content of the page, it says the Presenters are Paul and Mikhail.

Screen Shot 2020-04-15 at 12 37 08 PM

Comment (zchase) [reply content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md:10]: Yeah, there was a typo in the directory name for the page. I fixed the typo, thanks for catching!

Comment (zchase) [reply content/webinars/infrastructure-as-code-workshop-2020-04-09/index.md:10]: Fixed the presenters to match the video. I will still redirect them to the "gated" version on the page so we can try to capture the lead.

**Pull request merged**